code/game/machinery/turrets.dm

```diff
@@ -27,11 +27,14 @@
 	var/wasvalid = 0
 	var/lastfired = 0
 	var/shot_delay = 30 //3 seconds between shots
+	var/fire_twice = 0
+
 	use_power = 1
 	idle_power_usage = 50
 	active_power_usage = 300
 //	var/list/targets
 	var/atom/movable/cur_target
+	var/atom/movable/prev_target
```

Comment:
Looks like this is never cleared, which will prevent the target from soft-deling.

Reply:
I forgot to delete this variable, it doesnt do anything.